### PR TITLE
Fix/pr changes

### PR DESCRIPTION
**Pull request by JortWillemsen**
Some fixes for easy changes requested for the PR with stryker-dashboard.

I did not really look for formatting since we probably need to do that all at once when all other changes are completed.

### CODE REVIEW

File: packages/ui-components/src/components/hero-component.ts [hunk 1]
```diff
@@ -123,8 +123,12 @@ export class HeroComponent extends BaseElement {
               See your reports from anywhere
             </p>
             <div class="flex justify-center space-x-4 my-8">
-              <sme-button @click="${this.#handlePrimary}">Get started</sme-button>
-              <sme-button @click="${this.#handleSecondary}" priority="secondary">What is Stryker?</sme-button>
+              <sme-button @click="${this.#handlePrimary}"
+                >Get started</sme-button
+              >
```
Comment (xandervedder) [inline packages/ui-components/src/components/hero-component.ts:128]: Not necessarily for this PR, but I really don't like this formatting... 🙃

Comment (xandervedder) [reply packages/ui-components/src/components/hero-component.ts:128]: I guess increasing the width to something higher than 80 (?) would probably suffice, but we can do that later (in the actual repository).

File: packages/ui-components/src/components/top-bar.ts [hunk 2]
```diff
@@ -8,7 +8,7 @@ export class TopBar extends BaseElement {
 
   getLogo() {
     if (this.logo) {
-      return html`<img class="p-2 h-12" src="${this.logo}" />`;
+      return html``;
     } else {
       return html``;
     }
```
Comment (xandervedder) [inline packages/ui-components/src/components/top-bar.ts]: Remove this method as it's not being used anymore.

**Review by xandervedder — CHANGES_REQUESTED** [review]

File: packages/ui-components/src/components/top-bar.ts [hunk 3]
```diff
@@ -18,9 +18,14 @@ export class TopBar extends BaseElement {
     return html` <div
       class="text-white items-center bg-neutral-800 h-16 p-2 text-l flex flex-row"
     >
-      ${this.getLogo()}
+      ${this.logo && html`<img class="p-2 h-12" src="${this.logo}" />`}
```
Comment (jensk-dev) [inline packages/ui-components/src/components/top-bar.ts:14]: You could consider using lit's [`nothing`](https://lit.dev/docs/templates/conditionals/#conditionally-rendering-nothing) sentinel value, here.

```suggestion
      ${!!this.logo ? html`<img class="p-2 h-12" src="${this.logo}" />` : nothing}
```

Comment (xandervedder) [reply packages/ui-components/src/components/top-bar.ts:14]: I would personally only do this if the check gets more complex. For example: check if the logo is an empty string. Stuff like that.

**Review by xandervedder — APPROVED** [review]

**Pull request merged**